ethereum/vyper
# Faq - Added some basic questions

### - What I did

Added some initial questions/answers to the FAQ.

### - How I did it

Reading existing docs and issues. Built HTML to confirm output. 

### - How to verify it

Build docs and look at Frequently Asked Questions.  I'm wondering if the Vyper or Solidity question should include something explicit about why you'd use Vyper rather than Solidity - let me know what you think. 

### - Description for the changelog

Added some basic frequently asked questions. 

### - Cute Animal Picture

> put a cute animal picture here.
![image](https://user-images.githubusercontent.com/40311741/43421886-5d2c58fa-948b-11e8-8a81-72dcd4b1dbd9.png)
